test/transformer.spec.js

```diff
@@ -519,4 +519,54 @@ describe( 'transformer', () => {
         } );
     } );
 
+    describe( 'transform range widget', () => {
+        const xform = fs.readFileSync( './test/forms/widgets.xml', 'utf8' );
+        const result = transformer.transform( { xform } );
+
+        it( 'with empty appearance', () => {
+            return result.then( res => {
+                let doc = parser.parseFromString( res.form, 'text/xml' );
+                return Promise.all( [
+                    expect( doc.getElementsByTagName( 'input' )[ 91 ].getAttribute( 'name' ) ).to.equal( '/widgets/range_widgets/default' )
```

Comment:
If you had included a dedicated range.xml file the input[XX] check would be fine, but in the large widgets.xml file it's better not to do this. However, the tests are good to have with a small tweak. I think just replacing the `expect` statement with the following, makes it useful:

```js
expect( doc.querySelector[name="/widgets/range_widgets/default"]).type).to.equal('range')
```

This goes for all expect statements below (except the picker will be different as you know)

If you feel like splitting the range questions into a range.xml form anyway, that would be great, but up to you.

Reply:
I tried to use 

```js
expect( doc.querySelector[name="/widgets/range_widgets/default"].type).to.equal('range')
```
but it's look like the querySelector result is undefined, I though it's function(by adding (... ) ) but is not a function. 

Also, I tried to separate to small forms but not sure how to match it, the range-picker is input and select-minimal is select1 just the form UI is the same.

Reply:
No worries, I'll try the form matching test, and if successful will create a PR for it, that you can then review.